requirements.txt

```diff
@@ -1,8 +1,9 @@
-scikit-learn>=0.20
+scikit-learn==0.20.4
 pandas>=0.23
 numpy
 scipy
 statsmodels
 lockfile
 decorator
 networkx
+graphviz
```

Comment:
no need to have graphviz in the main dependencies (we don't want to), graphviz is optional and importing aikit should work without having graphviz installed (it's only used for some functions like pipeline visualization)

Reply:
ok, removed